tensorflow/lite/micro/tools/make/kissfft_download.sh

```diff
@@ -17,7 +17,7 @@
 # Called with following arguments:
 # 1 - Path to the downloads folder which is typically
 #     tensorflow/lite/micro/tools/make/downloads
```

Comment:
update to `${TENSORFLOW_ROOT}tensorflow/lite/micro/tools/make/downloads`?

Reply:
done